From 64d822a598f46f9d08a41eab6a7d416a6c54927b Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Date: Wed, 8 Jun 2022 16:03:49 +0900
Subject: [PATCH] Disable audio_thread_priority default features

with_dbus is supposed to be optional in gecko, but in practice, the
audio_thread_priority defaults force it on.
# https://github.com/kinetiknz/audioipc/pull/5/
---
 audioipc/Cargo.toml | 1 +
 client/Cargo.toml   | 1 +
 server/Cargo.toml   | 1 +
 3 files changed, 3 insertions(+)

diff --git a/third_party/rust/audioipc/Cargo.toml b/third_party/rust/audioipc/Cargo.toml
index d69c56d..8a9dd3d 100644
--- a/third_party/rust/audioipc/Cargo.toml
+++ b/third_party/rust/audioipc/Cargo.toml
@@ -30,6 +30,7 @@ memmap2 = "0.2"
 
 [target.'cfg(target_os = "linux")'.dependencies.audio_thread_priority]
 version = "0.26.1"
+default-features = false
 
 [target.'cfg(windows)'.dependencies]
 mio = "0.6.19"
diff --git a/third_party/rust/audioipc-client/Cargo.toml b/third_party/rust/audioipc-client/Cargo.toml
index 3ffb268..5dbf5f6 100644
--- a/third_party/rust/audioipc-client/Cargo.toml
+++ b/third_party/rust/audioipc-client/Cargo.toml
@@ -19,3 +19,4 @@ tokio = { version="0.1", default-features=false, features = ["rt-full"] }
 
 [dependencies.audio_thread_priority]
 version = "0.26.1"
+default-features = false
diff --git a/third_party/rust/audioipc-server/Cargo.toml b/third_party/rust/audioipc-server/Cargo.toml
index 2c82fbf..b2b710c 100644
--- a/third_party/rust/audioipc-server/Cargo.toml
+++ b/third_party/rust/audioipc-server/Cargo.toml
@@ -24,3 +24,4 @@ default-features = false
 
 [dependencies.audio_thread_priority]
 version = "0.26.1"
+default-features = false

